File: Exercise1/Exercise1aINTELLIJ/src/xmlrpcserwer/MyData.java
package xmlrpcserwer;

import java.net.InetAddress;
import java.net.UnknownHostException;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class MyData {
    public static void info() {


        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
        System.out.println("yyyy/MM/dd HH:mm:ss-> "+dtf.format(LocalDateTime.now()));

        System.out.println("Data: Alberto Hernandez Lado");

        System.out.println("User Name: " + System.getProperty("user.name"));

        System.out.println("Operating System Name: " + System.getProperty("os.name"));

        System.out.println("Version: " + System.getProperty("java.runtime.version"));

        InetAddress ip;
        try {
            ip = InetAddress.getLocalHost();
            System.out.println("Ip Address:" + ip);
        } catch (UnknownHostException e) {
            e.printStackTrace();
        }
    }
}

File: 2-Hernandez-Alberto-Lopez-Alvaro/Exercise 2b/gRPCclient/MyData.cs
// Include namespace system
using System;
using System.Net;

public class MyData
{
    public static void info()
    {
        var dt = DateTime.Now.ToString("yyyy/MM/dd");
        Console.WriteLine("yyyy/MM/dd " + dt);
        Console.WriteLine("Data:Alvaro Lopez Pereda and Alberto Hernandez Lado");
        Console.WriteLine("UserName: {0}", Environment.UserName);
        Console.WriteLine("Operating System Name:" + Environment.MachineName);
        String strHostName = string.Empty;
        IPHostEntry ipEntry = Dns.GetHostEntry(Dns.GetHostName());
        IPAddress[] addr = ipEntry.AddressList;

        for (int i = 0; i < addr.Length; i++)
        {
            Console.WriteLine("IP Address {0}: {1} ", i, addr[i].ToString());
        }
    }
}